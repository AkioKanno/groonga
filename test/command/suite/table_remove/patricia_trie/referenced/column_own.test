table_create Names TABLE_PAT_KEY ShortText
column_create Names nick_name COLUMN_SCALAR Names

table_remove Names

dump
