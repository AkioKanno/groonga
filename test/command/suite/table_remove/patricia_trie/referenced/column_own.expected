table_create Names TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Names nick_name COLUMN_SCALAR Names
[[0,0.0,0.0],true]
table_remove Names
[
  [
    [
      -2,
      0.0,
      0.0
    ],
    "[table][remove] a column that references the table exists: <Names.nick_name> -> <Names>"
  ],
  false
]
#|e| [table][remove] a column that references the table exists: <Names.nick_name> -> <Names>
dump
table_create Names TABLE_PAT_KEY ShortText
column_create Names nick_name COLUMN_SCALAR Names

